src/utilities.cc

```diff
@@ -60,7 +60,7 @@ using std::string;
 
 _START_GOOGLE_NAMESPACE_
 
-static const char* g_program_invocation_short_name = NULL;
+const char* g_program_invocation_short_name = NULL;
```

Comment:
Removing `static` is a bad idea. Can't you just move the definition of `IsGoogleLoggingInitialized` a couple of lines to the top above `glog_internal_namespace_`? Only the declaration would need to be moved to `logging.h.in`.

Reply:
Is it counter-intuitive to have decl in logging.h and def in utilities.cc, when both of them have their .h/.cc counterpart?

Reply:
Yes. However, the alternative would be to move all related functionality to `logging.cc` which I'd like to avoid at this point. For consistency, one can declare the function twice both in `logging.h.in` and `utilities.h`. But this is not really necessary.

Reply:
Updated